From b4cf064872b813beb1dc4d4f9d369ecbf4b599e1 Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Tue, 19 Aug 2014 19:30:44 -0400
Subject: [scsi] qla4xxx: Change default capture to firmware defined capture mask

Message-id: <1408476658-5056-7-git-send-email-cdupuis@redhat.com>
Patchwork-id: 88355
O-Subject: [PATCH RHEL 7.1 e-stor 06/20] qla4xxx: Change default capture to firmware defined capture mask
Bugzilla: 1089349
RH-Acked-by: Rob Evers <revers@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Tej Parkash <tej.parkash@qlogic.com>

scsi commit id a511b4afcb4f0fbe5e66bcc9dc07eb53d3510160

Signed-off-by: Tej Parkash <tej.parkash@qlogic.com>
Signed-off-by: Vikas Chaudhary <vikas.chaudhary@qlogic.com>
Reviewed-by: Mike Christie <michaelc@cs.wisc.edu>
Signed-off-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/qla4xxx/ql4_os.c b/drivers/scsi/qla4xxx/ql4_os.c
index cc8fbb3..596b3dc 100644
--- a/drivers/scsi/qla4xxx/ql4_os.c
+++ b/drivers/scsi/qla4xxx/ql4_os.c
@@ -83,11 +83,11 @@ MODULE_PARM_DESC(ql4xsess_recovery_tmo,
   " Target Session Recovery Timeout.\n"
   "\t\t  Default: 120 sec.");
 
-int ql4xmdcapmask = 0x1F;
+int ql4xmdcapmask = 0;
 module_param(ql4xmdcapmask, int, S_IRUGO);
 MODULE_PARM_DESC(ql4xmdcapmask,
    " Set the Minidump driver capture mask level.\n"
-   "\t\t  Default is 0x1F.\n"
+   "\t\t  Default is 0 (firmware default capture mask)\n"
    "\t\t  Can be set to 0x3, 0x7, 0xF, 0x1F, 0x3F, 0x7F");
 
 int ql4xenablemd = 1;
-- 
1.7.1